django/core/management/commands/test.py

```diff
@@ -79,8 +85,11 @@ def handle(self, *test_labels, **options):
             os.environ['DJANGO_LIVE_TEST_SERVER_ADDRESS'] = options['liveserver']
             del options['liveserver']
 
+
+        tags = set(options['tags']) if options['tags'] is not None else set()
+        exclude_tags = set(options['exclude_tags']) if options['exclude_tags'] is not None else set()
```

Comment:
And similar here. Note that since we are now using the `append` action, I think the default value from argparse is `[]` not `None`, so I don't believe we have to defend against `None` anymore.